src/main/java/kdt/vouchermanagement/Menu.java

```diff
@@ -13,6 +19,9 @@ public enum Menu {
     }
 
     public static Menu from(String input) {
-        return null;
+        Optional<Menu> foundMenu = Arrays.stream(values())
+                .filter(o -> Objects.equals(o.menu, input))
+                .findFirst();
+        return foundMenu.orElseThrow(() -> new InvalidMenuException("입력한 메뉴값이 유효하지 않습니다."));
```

Comment:
```suggestion
        Optional<Menu> foundMenu = Arrays.stream(values())
                .filter(o -> Objects.equals(o.menu, input))
                .findFirst()
                .orElseThrow(() -> new InvalidMenuException("입력한 메뉴값이 유효하지 않습니다."));
```

여기는 물론 취향차이긴 할 수도 있지만 이렇게 합쳐서 쓰는게 더 가독성이 좋아보일 것 같아요